datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/mirrormaker/KafkaMirrorMakerConnector.java

```diff
@@ -51,6 +91,14 @@ public KafkaMirrorMakerConnector(String connectorName, Properties config, String
         clusterName, LOG);
     _isFlushlessModeEnabled =
         Boolean.parseBoolean(config.getProperty(IS_FLUSHLESS_MODE_ENABLED, Boolean.FALSE.toString()));
+    _partitionFetchIntervalMs = Long.parseLong(config.getProperty(PARTITION_FETCH_INTERVAL,
+        Long.toString(DEFAULT_PARTITION_FETCH_INTERVAL)));
+    VerifiableProperties verifiableProperties = new VerifiableProperties(config);
+    _consumerProperties = verifiableProperties.getDomainProperties(KafkaBasedConnectorConfig.DOMAIN_KAFKA_CONSUMER);
+    _listenerConsumerFactory = new KafkaConsumerFactoryImpl();
+    _enablePartitionAssignment = _config.getEnablePartitionAssignment();
```

Comment:
Can we log the configs that are getting passed to the connector or atleast the enablePartitionAssignment config which is a critical config that decides how this connector behaves?

Reply:
Sure.